src/applications/EGFRvIIIIndexPredictor.h

```diff
@@ -748,11 +745,11 @@ VectorDouble  EGFRvIIIIndexPredictor::LoadTestData(const typename ImageType::Poi
   atlimIt.GoToBegin();
   while (!atlimIt.IsAtEnd())
   {
-    if (atlimIt.Get() == CAPTK::GLISTR_OUTPUT_LABELS::TUMOR)
-      etumorIndices.push_back(atlimIt.GetIndex());
-    else if (atlimIt.Get() == CAPTK::GLISTR_OUTPUT_LABELS::NONENHANCING)
-      necoreIndices.push_back(atlimIt.GetIndex());
-    else if (atlimIt.Get() == CAPTK::GLISTR_OUTPUT_LABELS::EDEMA)
+    //if (atlimIt.Get() == CAPTK::GLISTR_OUTPUT_LABELS::TUMOR)
```

Comment:
A little confused as to why TU and NE classes have been removed.

Reply:
i think it was just the naming convention that we used TUMOR and NONENHANCING instead of TU and NE. we did that long ago.